README.md

```diff
@@ -29,10 +29,16 @@ The user workload is typically on a Vertex AI notebook, so users can connect to
 ### Install a released version
 To install the latest released version of PathwaysJob version on your cluster, run the following command:
 ```sh
-VERSION=v0.1.0
+VERSION=v0.1.1
 kubectl apply --server-side -f https://github.com/google/pathways-job/releases/download/$VERSION/install.yaml
 ```
 
+To uninstall the latest released version of PathwaysJob version on your cluster, run the following command:
+```sh
+VERSION=v0.1.1
```

Comment:
We'll need to update this piece at every version bump. Is there a way to query our releases page for latest instead? I think jobset did that in xpk at some point.